.github/workflows/linux-eic-shell.yml

```
@@ -83,8 +83,16 @@ jobs:
         platform-release: "${{ env.platform-release }}"
         run: |
           # install this repo
-          CC="${{ matrix.CC }}" CXX="${{ matrix.CXX }}" CXXFLAGS="${{ matrix.CXXFLAGS }}" cmake -B build -S . -DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache -DCMAKE_BUILD_TYPE=${{ matrix.CMAKE_BUILD_TYPE }} -DUSE_ASAN=ON -DUSE_TSAN=OFF -DUSE_UBSAN=ON
+          ccache -z
+          export CC="${{ matrix.CC }}" CXX="${{ matrix.CXX }}" CXXFLAGS="${{ matrix.CXXFLAGS }}"
+          cmake -B build -S . \
+            -DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
+            -DCMAKE_BUILD_TYPE=${{ matrix.CMAKE_BUILD_TYPE }} \
+            -DUSE_ASAN=ON -DUSE_TSAN=OFF -DUSE_UBSAN=ON
+          export CCACHE_DEBUG=1 CCACHE_DEBUGDIR=$PWD/ccache-debug
           cmake --build build -- -j 2 install
+          find $CCACHE_DEBUGDIR ! -name "*.ccache-log" -delete
```

Only attempt to delete files...
```suggestion
          find $CCACHE_DEBUGDIR ! -name "*.ccache-log" -type f -delete
```